File: go/1015.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c, d float64
	fmt.Scanln(&a, &c)
	fmt.Scanln(&b, &d)
	distance := math.Sqrt((b-a)*(b-a) + (d-c)*(d-c))
	fmt.Printf("%.4f", distance)
}

File: python/1015.py
from math import sqrt
a, c = input().split(" ")
b, d = input().split(" ")
a, b, c, d = float(a), float(b), float(c), float(d)

distance = sqrt(((b-a)*(b-a)+(d-c)*(d-c)))
print(f"{distance:.4f}")
